packages/plugins/src/plugins/diagnostic-fix.plugin.ts

```diff
@@ -37,6 +38,8 @@ export class DiagnosticFixPlugin implements Plugin<DiagnosticFixPluginOptions> {
     2612, // addMissingDeclareProperty
   ];
 
+  attemptedToFix: string[] = [];
```

Comment:
We should reset this array on either line 90 or line 44 so we don't retain the hashes for entirety of the execution. This could lead to memory issues in really large applications if we don't

Reply:
Thanks. Done.